manager/latest-mac.yml

```diff
@@ -1,19 +1,17 @@
-version: 1.9.0
+version: 1.10.0
 files:
-  - url: Outline-Manager.zip
-    sha512: RsMSpb6tBzdSBMQK4XZrsgsdj4VijdeeOmbci8E37698R5tvbGru8xLUAYTAz4EW4RvGvJg2AqumFW4qnYLaNg==
-    size: 98350290
-    blockMapSize: 104655
-  - url: Outline-Manager.dmg
-    sha512: Mo8Fd/hsxFxTKZGddd6W4P9V+CQ8GleJKXkCRkSZ90rpFNxCjrcmcf/RiR3CFZOubYbqA0CjOlz+2HxKyvBNZw==
-    size: 101702614
-  - url: Outline-Manager.dmg
-    sha512: Mo8Fd/hsxFxTKZGddd6W4P9V+CQ8GleJKXkCRkSZ90rpFNxCjrcmcf/RiR3CFZOubYbqA0CjOlz+2HxKyvBNZw==
-    size: 101702614
-  - url: Outline-Manager.dmg
-    sha512: Mo8Fd/hsxFxTKZGddd6W4P9V+CQ8GleJKXkCRkSZ90rpFNxCjrcmcf/RiR3CFZOubYbqA0CjOlz+2HxKyvBNZw==
-    size: 101702614
-path: Outline-Manager.zip
-sha512: RsMSpb6tBzdSBMQK4XZrsgsdj4VijdeeOmbci8E37698R5tvbGru8xLUAYTAz4EW4RvGvJg2AqumFW4qnYLaNg==
-releaseDate: '2021-09-28T16:43:10.382Z'
-stagingPercentage: 100
+  - url: https://s3.amazonaws.com/outline-releases/manager/macos/1.10.0/1/Outline-Manager.zip
+    sha512: djBMduCWhKz6F833LUFvXhsK1DPdhucPWgRTqbO7mBKg+EZDtqKBrd7t7KxQmoQp3QMEQJLqXQHdzY6YjhKmew==
+    size: 99404940
+  - url: https://s3.amazonaws.com/outline-releases/manager/macos/stable/Outline-Manager.dmg
```

Comment:
Add back these duplicated entries for backward compatibility. They seem to be related to different CPU architectures, should be able to remove them by introducing the universal architecture.